docker-compose.yml

```diff
@@ -45,3 +45,12 @@ redis:
     - "6379"
   volumes:
     - redisdata:/data
+
+test:
```

Comment:
I think it would be better to keep a separate docker-compose specifically for the tests. Make a copy and maybe even keep it in the ../tests directory? Or at least name it with a -test suffix.

Reply:
Fixed